sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSDefaultParameterValueConstants.java

```diff
@@ -11,4 +11,8 @@ public class PDSDefaultParameterValueConstants {
     public static final int DEFAULT_TIME_TO_WAIT_IN_MILLISECONDS_FOR_SCRIPT_CANCELLATION_CHECK = MINIMUM_TIME_TO_WAIT_IN_MILLISECONDS_FOR_SCRIPT_CANCELLATION_CHECK;
     public static final int MAXIMUM_TIME_TO_WAIT_IN_MILLISECONDS_FOR_SCRIPT_CANCELLATION_CHECK = 5000; // 5 seconds
 
+    public static final int DEFAULT_MINUTES_TO_WAIT_FOR_PRODUCRESULT = 60 * 2; // 2 hours
```

Comment:
I changed to `DEFAULT_MINUTES_TO_WAIT_FOR_PRODUCT` because fields, other related parts do not use "result" anywhere - so it is better (but I added the "t" - thanx)